eternalcore-core/src/main/java/com/eternalcode/core/command/argument/SpeedTypeArgument.java

```diff
@@ -0,0 +1,37 @@
+package com.eternalcode.core.command.argument;
+
+import com.eternalcode.core.notification.Notification;
+import com.eternalcode.core.translation.Translation;
+import com.eternalcode.core.translation.TranslationManager;
+import com.eternalcode.core.feature.essentials.speed.SpeedType;
+import com.eternalcode.core.viewer.BukkitViewerProvider;
+import dev.rollczi.litecommands.argument.ArgumentName;
+import dev.rollczi.litecommands.command.LiteInvocation;
+import dev.rollczi.litecommands.suggestion.Suggestion;
+import panda.std.Option;
+import panda.std.Result;
+
+import java.util.Arrays;
+import java.util.List;
+
+@ArgumentName("type")

```

Comment:
Zmieniłbym nazwę tego argumentu np. na "speedType"

Reply:
Zaznaczę, że jest display name argumentu, adnotacja trochę nie fortunnie nazwana. 

w komendzie będzie to dziwnie wyglądać:
/speed <speedType>

/speed <type> 
jest w pełni zrozumiałe w obecnym kontekście